{{cookiecutter.project_slug}}/{{cookiecutter.project_module}}/app.py

```diff
@@ -42,17 +42,12 @@ def init_app(application, interface):
     else:
         from {{cookiecutter.project_module}}.settings import Development
         application.config.from_object(Development())
-    # Configure the logger and handler.
-    application.logger.setLevel(application.config["LOGLEVEL"])
-    formatter = logging.Formatter("[%(levelname)s] [%(name)s] %(message)s")
-    for handler in application.logger.handlers:
-        handler.setFormatter(formatter)
-    # Set the logging level and add handlers to desired packages here.
-    # FIXME: Remove those that are too spammy and uninteresting.
-    cors_logger = logging.getLogger("flask_cors")
-    cors_logger.setLevel(application.config["LOGLEVEL"])
-    for handler in application.logger.handlers:
-        cors_logger.addHandler(handler)
+    # Configure logging
+    # The flask logger, when created, disables existing loggers. The following
+    # statement ensures the flask logger is created, so that it doesn't disable
+    # our loggers later when it is first accessed.
```

Comment:
Are you sure about this? I would have thought the flask logger is already created at this point since we're passing in the application instance.

Reply:
Yes, the [creation is deferred](https://github.com/pallets/flask/blob/f808c20139649b747f604492bc33b61a7dd3e13a/flask/app.py#L612) until it is first accessed. I discovered during testing and noticed that flask log statements weren't formatted as expected, adding this statement fixes it. [This will change in the next Flask version](http://flask.pocoo.org/docs/dev/api/#flask.Flask.logger).